.editorconfig

```diff
@@ -0,0 +1,11 @@
+# http://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
```

Comment:
@paolobueno we havnt agreed on using it :D but its fairly standard format - spaces, 2 spc indent rest is to please eslint.
Would you like to change/add options?